docs/content/en/docs/references/deprecation.md

```diff
@@ -70,6 +70,8 @@ Exceptions will always be announced in all relevant release notes.
 
 ## Current deprecation notices
 
+04/07/2021: Release v1.22.0 deprecates the `--render-only` and `--render-output` flags from `skaffold run`, in favor of simply using the `skaffold render` command directly. 
```

Comment:
This change is (and should be) deprecating their use on `skaffold dev` too.

Reply:
i just realized the `--render-only` flag existed on both commands, but the `--render-output` only existed on run 🤷  either way good call out, updated